_episodes/04-pipefilter.md

```diff
@@ -100,7 +99,7 @@ $ wc *.pdb
 
 > ## Using Wildcards
 >
-> When run in the `molecules` directory, which `ls` command(s) will
```

Comment:
Original suggests that more than one answer may be correct. Modified suggests only one can be correct.

Reply:
Ahh okay. For some reasons I was thinking about `ls`command being singular (there's only one command) but not about there being multiple answers.